### ISSUE TITLE
NTR: MRI or CT: basilar meningeal enhancement (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8942]
New term request MRI or CT: basilar meningeal enhancement (samant_101125022405): Gadolinium included; basal cistern

**Comment by pnrobinson** [comment]
@MickeySegal
I may not be familiar with the terminology, but because of the order of this item I think it is related to tuberculosis and wonder if this might be "Basal" instead of "Basilar"
basal meningeal enhancement 
https://pubmed.ncbi.nlm.nih.gov/17036236/. "The classic finding in TBM is marked contrast enhancement outlining the basal cisterns."
basilar sound too much like basilar artery?

**Comment by MickeySegal** [comment]
Both basal and basilar are used, e.g., https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121769/ "While TBM causes predominantly basilar meningeal enhancement"

Basilar sounds more baroque, but since it refers to the basal cisterns (https://link.springer.com/article/10.1007/s00330-022-08911-3), it may be best to choose one of these:

- basilar meningeal enhancement
- basal cistern meningeal enhancement

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000851
- label: Basilar meningeal enhancement
- parents: [Abnormal meningeal morphology (HP:0010651)](https://hpo.jax.org/app/browse/term/HP:0010651)
- pmid: 
- def: Abnormal enhancement of CT or MIR contrast medium in the basilar meninges, that is, those located in the basal cisterns or spaces at the base of the brain, due to increased permeability of blood vessels in these areas.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8942
- synonyms:
